### ISSUE TITLE
More LibGen sections

### ISSUE THREAD

**Issue by manuelvargastapia** [manuelvargastapia/libgen_mobile_app#14]
By now, the app only provides access to the Sci-Tech section. Access to other sections (Fiction, Comics, etc) is desired.

**Comment by manuelvargastapia** [comment]
UPDATE: the API now supports the Fiction section and the front end is in development. Hopefully, this feature will be available on the next release, plus support for several languages.

**Comment by manuelvargastapia** [comment]
UPDATE: in the upcoming version 1.1.0, the Fiction section is supported by the app.

**Comment by PrithirajN** [comment]
Yes, addition of comics and magazine will make this app an one-stop for all types of books.
Also, if the home page of the app could show the recently uploaded books(different for different sections) then it would be great.
